### DISCUSSION TITLE
Contribute outside GSoC

### DISCUSSION THREAD

**Discussion by ricard1997** [MDAnalysis/mdanalysis#4551] [GSoC Discussions]
Hi all, I have just read that I missed the deadline for the pre-proposal. However, I have some code I developed regarding analysis of membranes that I would like to be included in MDAnalysis. Is there someone I can talk about this project? I do not need this to be part of GSoC, just would like to contribute to this software that has helped me a lot during my PhD.

**Comment by hmacdope** [comment]
@ricard1997 you are more than welcome to contribute to MDAnalysis! You are welcome to contribute to the main repo (see contributing [guide](https://userguide.mdanalysis.org/stable/contributing.html)), but it sounds like your work may be well suited to an [MDAKit](https://mdakits.mdanalysis.org/about.html), a kind of plugin that houses specialist analysis.

**Reply by orbeckst** [reply]
There's a whole part of the [discussion forum dedicated to MDAKits](https://github.com/MDAnalysis/mdanalysis/discussions/categories/mdakit-discussions) (and there's also a [#MDAKits discord channel](https://discord.com/channels/807348386012987462/1063549680090357890). 

We have a [tutorial on making an MDAKit](https://mdakits.mdanalysis.org/makingakit.html) and an overview video [Making an MDAKit - a short introduction](https://www.youtube.com/watch?v=viCPUHkgSxg&t=221s).

To learn more about MDAKits and our vision for more sustainable community developed molecular simulation tools, please see our [SciPy proceedings paper](https://conference.scipy.org/proceedings/scipy2023/ian_kenney.html):

* Irfan Alibay, Lily Wang, Fiona Naughton, Ian Kenney, Jonathan Barnoud, Richard J Gowers, and Oliver Beckstein. MDAKits: A framework for FAIR-compliant molecular simulation analysis . In Meghann Agarwal, Chris Calloway, and Dillon Niederhut, editors, Proceedings of the 22nd Python in Science Conference (SCIPY 2023), pages 76–84, Austin, TX, 2023. doi: [10.25080/gerudo-f2bc6f59-00a](https://doi.org/10.25080/gerudo-f2bc6f59-00a).

**Reply by ricard1997** [reply]
Thanks! Ill take a look and hope to collaborate soon
